docs/DEVELOPMENT.md

```diff
@@ -51,63 +53,209 @@ You can redeploy all components with kubectl and ko. Just make sure to
 [configure ko with kind](https://github.com/google/ko/blob/main/README.md#with-kind).
 
 ```sh
-$ export KO_DOCKER_REPO=kind.local
-$ kubectl kustomize ./config | ko apply -f -
```

Comment:
Now we can just copy paste. This $ was too annoying.

Reply:
Yess 😄